src/xdocs/config_javadoc.xml

```diff
@@ -1210,6 +1210,13 @@ public boolean isSomething()
             <td><code>false</code></td>
             <td>5.1</td>
           </tr>
+          <tr>
+            <td>allowedAnnotations</td>
+            <td>List of annotations that could allow missed documentation.</td>
```

Comment:
could allow? Are you not sure?
Be assertive in documentation. Remove the word `could`.

Reply:
fixed